addons/payment/models/payment_acquirer.py

```diff
@@ -442,9 +444,11 @@ def _get_default_partner_country_id(self):
     partner_phone = fields.Char('Phone')
     html_3ds = fields.Char('3D Secure HTML')
 
-    callback_eval = fields.Char('S2S Callback', help="""\
-        Will be safe_eval with `self` being the current transaction. i.e.:
-            self.env['my.model'].payment_validated(self)""", oldname="s2s_cb_eval", groups="base.group_system")
+    callback_ir_model_id = fields.Many2one('ir.model', 'Callback Document Model', groups="base.group_system")
```

Comment:
`callback_model_id` should suffice, it's not ambiguous and it's 3 char shorter (it's something...)